packages/api-client-core/src/GadgetFunctions.ts

```diff
@@ -162,6 +172,7 @@ export interface GetFunction<OptionsT, SelectionT, SchemaT, DefaultsT> {
   selectionType: SelectionT;
   optionsType: OptionsT;
   schemaType: SchemaT | null;
+  plan?: <Options extends OptionsT>(options?: LimitToKnownKeys<Options, OptionsT>) => GQLBuilderResult;
 }
 
 export interface GlobalActionFunction<VariablesT> {
```

Comment:
Should `GlobalActionFunction` get a `plan`?

Reply:
yup. good catch.